erts/emulator/beam/erl_bits.c

```diff
@@ -40,6 +40,17 @@
 
 #define BIT_IS_MACHINE_ENDIAN(x) (((x)&BSF_LITTLE) == BIT_ENDIAN_MACHINE)
 
+#if (SIZEOF__FLOAT16 == 2)
+typedef _Float16 erlfp16;
+#define FP16_FROM_FP64(x) ((_Float16) x)
+#define FP16_TO_FP64(x) ((double) x)
+#else
+typedef uint16_t erlfp16;
```

Comment:
This seems to break our Windows builds, can you change `uint16_t` to `Uint16`?

Reply:
Oops, fixed! I have also rebased to fix a tiny conflict on erl_lint.

Reply:
Perfect, thank you. :)